src/components/AccountWarning/CannedMessages.ts

```diff
@@ -128,7 +128,7 @@ We've noticed that you joined game #{{game_id}} but didn't make any moves.
 
 It's possible you didn't notice this game start - we understand that.
 
-However, could you please take care avoid this situation, so that other users are not left waiting and wondering.
+If you can be alert for this, so that other users are not left waiting and wondering, that would be great..
```

Comment:
![office_space.gif](https://media1.tenor.com/m/o6mL88Hz5msAAAAC/that-would-be-great-office-space.gif)

Reply:
Contributions welcome :p     (I was about to post a link to the code ... 🤦‍♀️  )

Reply:
Yeah.   Anoek hunted again for ways to fix the problem, not the symptom (how to help people not miss games that start), but we don't have it.

Mind you, IIRC folk have suggested that "having multiple ways to join a game open at once" is a contributor.   I don't recall the details - @anoek  ?

In the mean time, we have to tone down 'cause folk are getting riled.  And it happens a _lot_, so that's a lot of opportunity for complaining.